openfin-csharp-api-test/openfin-csharp-api-test/OpenfinTests.cs

```diff
@@ -211,6 +212,44 @@ public async Task AppEventsInitiallyOpen()
             Assert.IsTrue(startedFired);
         }
 
+        private Dictionary<string, object> getProcessInfo()
+        {
+            string script = "return await fin.System.getProcessList()";
+            driver.ExecuteScript(script); // First call is different to following calls
```

Comment:
in what way?

Reply:
I think it sometimes returns the values for the last window location.  But I'm not sure and haven't investigated.
At this stage, I'm not even sure what "correct" values are, as I'm not confident what `getProcessList` should return nor that it ever actually returns them.

Reply:
Understood.